src/main/java/mate/academy/internetshop/controller/IndexController.java

```diff
@@ -8,9 +12,16 @@
 
 public class IndexController extends HttpServlet {
 
+    @Inject
+    private static ItemService itemService;
+
     @Override
     protected void doGet(HttpServletRequest req, HttpServletResponse resp)
             throws ServletException, IOException {
+        itemService.update(new Item(12L, "sobaka", 32.));
```

Comment:
@joej0e What is the id value in this case? 12? Are you able to update this row in the DB? Do you have the row with this id?

Reply:
Here, I am expecting to do next:
- create, get the item with autogenerated id as a response
- get, get the item by id
- update, update some field (for example, price)
- get, verify that data was updated in the DB
- delete, delete the item by id